### PR TITLE
Feature/aop

### PR DESCRIPTION
**Pull request by JohnBDonner**
1.0.0

### CODE REVIEW

File: lib/transaction_logger/transaction.rb [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
   attr_accessor :name
   attr_accessor :context
+  attr_accessor :level_threshold
+  attr_accessor :level_threshold_broken
 
   # @param parent [TransactionLogger::Transaction] Parent of transaction
   # @param lmbda [Proc] The surrounded block
   #
-  def initialize(parent=nil, lmbda)
+  def initialize(parent=nil, log_prefix, logger, level_threshold, lmbda)
```
Comment (sebastian-julius) [inline lib/transaction_logger/transaction.rb]:
```ruby
def initialize(options={}, lmbda)
@parent = options[:parent]
```

**Comment by JohnBDonner** [comment]
Should be good to merge :+1:

**Comment by sebastian-julius** [comment]
:+1:

**Comment by sebastian-julius** [comment]
#23 done

**Pull request merged**